portfolio/src/main/webapp/js/script.js

```diff
@@ -69,6 +69,24 @@ function getComments() {
     data.comments.forEach((line) => {
       commentsSection.appendChild(createComment(line, data));
     });
+
+    pageNum = 0;
```

Comment:
The pageNum is 0 here. So we will never reach the "else" branch of Line 75 ~ 82.

Reply:
I changed this, thank you!